tm-parsers/js/js.tm

```diff
@@ -781,7 +797,7 @@ Declaration<Yield, Await> -> Decl /* interface */:
   | AmbientDeclaration
 ;
 
-HoistableDeclaration -> Decl /* interface */:
+HoistableDeclaration<Await> -> Decl /* interface */:
```

Comment:
Actually this Await is not really necessary because it influences the function name identifier only (see my comment below).